static/js/click_handlers.js

```diff
@@ -1032,4 +1033,8 @@ export function initialize() {
     $(".settings-header.mobile .fa-chevron-left").on("click", () => {
         settings_panel_menu.mobile_deactivate_section();
     });
+
+    // Click handler for the deactivate organization button present in both
+    // personal settings and organization settings.
+    $("body").on("click", ".deactivate_realm_button", settings_org.deactivate_organization);
```

Comment:
I'd prefer independent calls from the initialization functions of each module; we're aiming to delete this file over time.